src/js/edu-benefits/1990e/config/form.js

```diff
@@ -31,7 +48,36 @@ const formConfig = {
       pages: {
         applicantInformation: {
           path: 'applicant-information',
-          title: 'Applicant information'
+          title: 'Applicant information',
+          initialData: {},
+          uiSchema: {
+            applicantFullName: fullName.uiSchema,
+            applicantSocialSecurityNumber: ssn.uiSchema,
+            applicantDateOfBirth: currentOrPastDate.uiSchema('Applicant date of birth'),
```

Comment:
@goldenmeanie Can you check over https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Products/Education%20Benefits/1990E/1990E-outline-draft.md and change anything that isn't exactly as it should appear on the form?